### PR TITLE
docs: add provider docs

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

**Comment by sebald** [comment]
@sarahgm hmmm. Weird error 🤔

File: docs/content/components/provider/provider.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      description: 'The children of the component.',
+    },
+    {
+      property: 'theme',
+      type: 'Theme',
+      default: 'none',
+      description: 'The use that should be used within the provider context.',
+    },
+    {
+      property: 'selector',
```
Comment (sebald) [inline docs/content/components/provider/provider.mdx:50]: there is no selector anymore :D

**Review by sebald — CHANGES_REQUESTED** [review]

File: docs/content/components/provider/provider.mdx (hunk 1 above)
Comment (sarahgm) [reply docs/content/components/provider/provider.mdx:50]: it is

Comment (sarahgm) [reply docs/content/components/provider/provider.mdx:50]: should I remove the prop?

**Review by OsamaAbdellateef — APPROVED** [review]

**Review by sebald — APPROVED** [review]

**Pull request merged**